.gitignore

```diff
@@ -133,4 +133,7 @@ venv.bak/
 /site
 
 # mypy
-.mypy_cache/
\ No newline at end of file
+.mypy_cache/
+
```

Comment:
Update the gitignore to avoid migrations from getting pushed

Reply:
Okay